docs/visualization/plotting-overview.ipynb

```diff
@@ -308,6 +308,58 @@
    "source": [
     "(100 * (da1 - da1.min())).plot(norm='log')"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Migrating to the New Backend\n",
+    "\n",
+    "Since version 0.17, Scipp can use either the old builtin plotting backend or [Plopp](https://scipp.github.io/plopp/).\n",
+    "The old backend is deprecated and will be removed in the first release in or after August 2023.\n",
+    "It is recommended to switch to the new backend already now to avoid breaking your code through an update later.\n",
+    "There are 3 ways of doing so:\n",
+    "\n",
+    "### Use Plopp Directly\n",
+    "\n",
+    "Instead of using `sc.plot(da)` or `da.plot()`, use\n",
+    "```python\n",
+    "import plopp as pp\n",
+    "pp.plot(da)\n",
+    "```\n",
+    "\n",
+    "### Patch Scipp Via Plopp\n",
+    "\n",
+    "Plopp provide `patch_scipp` to set Plopp as the default plotting backend in Scipp. (`unpatch_scipp` can be used to reset to the old backend.)\n",
+    "```python\n",
+    "import plopp as pp\n",
+    "pp.patch_scipp()\n",
+    "# Uses Plopp as backend:\n",
+    "sc.plot(da)  # or da.plot\n",
+    "```\n",
+    "\n",
+    "### Set a Configuration Value\n",
+    "\n",
+    "Set the `plot/use_plopp` configuration value to set Plopp as the default backend.\n",
+    "See [Runtime Configuration](../reference/runtime-configuration.rst).\n",
+    "\n",
+    "In your config file:\n",
+    "```yaml\n",
+    "plot:\n",
+    "  use_plopp: True\n",
+    "```\n",
+    "And then Scipp uses Plopp for all plotting:\n",
+    "```python\n",
+    "sc.plot(da)  # or da.plot\n",
+    "```\n",
+    "\n",
+    "### Using the Old Backend\n",
+    "\n",
+    "There are some changes to functionality in Plopp compared to the old backend.\n",
+    "Most notably, automatic resampling has been removed.\n",
+    "If the old backend is required, it is possible to use `scipp.legacy_plot` instead of `sc.plot` to use the old backend regardless of any configuration settings or patching through Plopp.\n",
+    "But use this at your own risk as `scipp.legacy_plot` will be removed in the future."
```

Comment:
I think I would just remove these two lines?

Reply:
I think it is  valuable to be able to force the old behaviour without any risk of config settings to mess up ones code.

Reply:
Ok, can you open an issue to remove these lines when we remove the deprecated plotting. Otherwise we will forget.

Reply:
https://github.com/scipp/scipp/issues/2879